core/pom.xml

```diff
@@ -28,6 +28,26 @@
                     </excludePackageNames>
                 </configuration>
             </plugin>
+
+            <plugin>
```

Comment:
Is it possible to perhaps put this config in the parent pom and have the children inherit?  Perhaps in a pluginManagement section?